.azure-devops/full-pipeline.yml

```diff
@@ -3,13 +3,52 @@
 # https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
 
 jobs:
-  - job: WindowsServer2019
+  - job: Windows_10_Latest
     pool:
-      name: OETest-OfficeClientApps
+      name: OE-OfficeClientApps
     steps:
       - template: ./install.yml
       - template: ./lint.yml
       - template: ./build.yml
       - template: ./devcerts.yml
       - template: ./edgewebview.yml
       - template: ./test.yml
+        parameters:
+          webView: "edge-chromium"
+
+  - job: WebView_EdgeLegacy
+    pool:
+      name: OE-OfficeClientApps
+    steps:
+      - template: ./install.yml
+      - template: ./lint.yml
+      - template: ./build.yml
+      - template: ./devcerts.yml
+      - template: ./edgewebview.yml
+      - template: ./test.yml
+        parameters:
+          webView: "edge-legacy"
+
+  # Need to fix tests (use of find-process module) to be able to support IE 11 webview
```

Comment:
I would prefer for you to just create a task to fix the tests in IE 11 and include this code and instructions in its description.
We can then not have dead comments in the code.